slides/bbs-efspi-mmrm-dec2022.qmd

```diff
@@ -0,0 +1,128 @@
+---
+title: "mmrm"
+author: "Daniel Sabanes Bove"
+institute: (Statistical Engineering,
+  Pharma Product Development Data Sciences, 
+  Roche)
+date: "@BBS/EFSPI Workshop 8 Dec 2022"
+format: 
+  revealjs:
+    incremental: true
+    logo: https://github.com/RConsortium/asa-biop-swe-wg/raw/main/sticker/sticker-1200.png
+    slide-number: c/t
+title-slide-attributes: 
+  data-background-image: https://github.com/openpharma/mmrm/raw/main/man/figures/logo.png
+  data-background-size: 20%
+  data-background-opacity: "0.5"
+  data-background-position: 80% 10%
+---
+
+## Acknowledgments
+
+::: nonincremental
+::: columns
+::: {.column width="50%"}
+Authors:
+
+-   Kevin Kunzmann (Boehringer Ingelheim)
+-   Brian Matthew Lang (MSD)
+-   Ya Wang (Gilead)
+-   Julia Dedic (Roche)
+-   Doug Kelkhoff (Roche)
+-   Liming Li (Roche)
+:::
+
+::: {.column width="50%"}
+Thanks to:
+
+-   Ben Bolker (McMaster University)
+-   Software Engineering working group (SWE WG)
+-   Gonzalo Duran Pacheco (Roche)
+-   Craig Gower-Page (Roche)
+-   Dinakar Kulkarni (Roche)
+-   Davide Garolini (Roche)
+:::
+:::
+:::
+
+## Motivation
+
+-   2 years ago, we thought we had solved the MMRM problem, using `lme4` and `lmerTest`
+-   However in March this year, Gonzalo and Julia told us the bad news: this approach failed on their Ophthalmology data sets (slow, did not converge)
+-   `nlme` does not give Satterthwaite adjusted degrees of freedom, has convergence issues, and with `emmeans` it is only approximate
+-   Next we tried to extend `glmmTMB` to calculate Satterthwaite adjusted degrees of freedom, but it did not work
+
+## Idea
+
+-   Because `glmmTMB` is always using a random effects representation, we cannot have a real unstructured model (uses $\sigma = \varepsilon > 0$ trick)
+-   We only want to fit a fixed effects model with a structured covariance matrix for each subject
+-   The idea is then to use the Template Model Builder (`TMB`) directly - as it is also underlying `glmmTMB` - but code the exact model we want
+-   We do this by implementing the log-likelihood in `C++` using the `TMB` provided libraries 
+
+## Advantages of `TMB`
+
+-   Since we use `C++` it is fast (but `Rcpp` could also have been used instead)
```

Comment:
Is that an "or"? Rcpp is C++ after all?

"fast C++ framework for defining likelihoods"